proto/inx.proto

```diff
@@ -212,6 +226,15 @@ message PayloadValidationResponse {
   string error = 2;
 }
 
+message RootBlocksResponse {
+  repeated RootBlock root_blocks = 1;
+}
+
+message RootBlock {
+  BlockId blockId = 1;
```

Comment:
Why not `block_id` ?

Reply:
Probably just copied it from `LedgerOutput`. Should we change it there as well?